### PR TITLE
Add Docker on prefixed/downgraded version only

### PR DESCRIPTION
**Pull request by TomasVotruba**
Rector is moving fully to downgraded/prefixed deploy in https://github.com/rectorphp/rector-prefixed
It will be the one and only available version.

Thus Docker will be build in there on downgraded and prefixed version :+1:

### CODE REVIEW

File: build/target-repository/Dockerfile [hunk 1]
```diff
@@ -31,23 +31,11 @@ RUN apt-get update && apt-get install -y \
...
-RUN wget https://github.com/humbug/php-scoper/releases/download/0.14.0/php-scoper.phar -N --no-verbose
-
 # This is to make parsing version possible
 COPY .git .git
 
-# First copy composer.json only to leverage the build cache (as long as not git-committing)
-COPY composer.json composer.json
-RUN composer install --no-dev --ansi
-
-# Add source and generate full autoloader
```
Comment (TomasVotruba) [inline build/target-repository/Dockerfile:44]: Composer is already dumped in prefixed version, so dump would override and break it

File: build/target-repository/Dockerfile [hunk 2]
```diff
@@ -31,23 +31,11 @@ RUN apt-get update && apt-get install -y \
...
+# Add source
 COPY . .
-RUN composer dump-autoload --classmap-authoritative --no-dev
-
-RUN rm -f "phpstan-for-rector.neon" \
-    && php -d memory_limit=-1 php-scoper.phar add-prefix preload.php bin config packages rules src templates vendor composer.json --output-dir /scoped --config scoper.php \
-    && composer dump-autoload --optimize --classmap-authoritative --no-dev --working-dir /scoped
```
Comment (TomasVotruba) [inline build/target-repository/Dockerfile:39]: The code is scoped and downgraded already :+1:

File: build/target-repository/Dockerfile [hunk 3]
```diff
@@ -63,7 +51,7 @@ ENTRYPOINT [ "rector" ]
 VOLUME ["/project"]
 WORKDIR "/project"
 
-COPY --from=build /scoped /rector
+COPY --from=build . /rector
```
Comment (TomasVotruba) [inline build/target-repository/Dockerfile:54]: I hope this is right :) could anyone check?

**Comment by TomasVotruba** [comment]
Hi @mpdude , could I ask to review this PR?

I got something wrong https://github.com/rectorphp/rector-prefixed/runs/2517927781

Not sure what the exit code means without any error message:

image

<br>

Thank you

**Comment by mpdude** [comment]
```
RUN mkdir -p /tmp/opcache \
    && /rector/bin/rector list \
    && chmod 777 -R /tmp
```

is trying to run `/rector/bin/rector`, but the binary is not in this place. 

Try moving the 

```
# Add source
COPY . /rector
```

lines after the `ENV PATH ...` statement.

**Comment by TomasVotruba** [comment]
I'll try to copy-paste it :) thanks

**Comment by TomasVotruba** [comment]
Here it is https://github.com/rectorphp/rector/commit/c900b9ccb065159aaa28802b9b9503c1728530ae

**Comment by TomasVotruba** [comment]
Wow, it's :heavy_check_mark: and running! https://github.com/rectorphp/rector-prefixed/runs/2531702003

You're magacian, thank you :+1:

**Pull request merged**